Ntopng-Release: 	3.4
Bug-Reference-URL:	https://github.com/ntop/ntopng/issues/1833

The open function in TimeSeriesExporter::createDump requires 3 arguments.
To prevent an 'open_missing_mode declared with attribute error:' on gcc-7.3,
a mode has been added to the open call

--- ntopng.orig/src/TimeSeriesExporter.cpp      2018-06-26 08:43:23.513583569 +0000
+++ ntopng/src/TimeSeriesExporter.cpp   2018-06-26 08:53:07.690044358 +0000
@@ -74,7 +74,7 @@
 #ifdef WIN32
   fd = open(fname, O_RDWR | O_CREAT | O_EXCL, _S_IREAD | _S_IWRITE);
 #else
-  fd = open(fname, O_RDWR | O_CREAT | O_EXCL | S_IREAD | S_IWRITE);
+  fd = open(fname, O_RDWR | O_CREAT, 0600 | O_EXCL | S_IREAD | S_IWRITE);
 #endif
 
   if(fd == -1)
